packages/flutter_image/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.0.2
```

Comment:
A new feature should be a minor version bump, so 4.1.0

Reply:
done.